tox.ini

```diff
@@ -31,11 +31,11 @@ deps =
   wheel
 commands =
   # safety measure to assure we do not accidentaly run tests with broken dependencies
+  # We add coverage options but not making them mandatory as we do not want to force
+  # pytest users to run coverage when they just want to run a single test with `pytest -k test`
   python -m pip check
   {envpython} -m pytest \
-  --cov "{envsitepackagesdir}/ansiblelint" \
-  --junitxml "{toxworkdir}/junit.{envname}.xml" \
-  {posargs:}
+  {posargs:--cov "{envsitepackagesdir}/ansiblelint" --junitxml "{toxworkdir}/junit.{envname}.xml" --cov-report term-missing:skip-covered --cov-report xml:.test-results/pytest/cov.xml --no-cov-on-fail}
```

Comment:
Oh, you changed the xml location, I'd like to keep it in the old one, the whole dir is gitignored

Reply:
Let's leave junit changes out and discuss separately, I guess.

Reply:
I removed that as that is outside the scope of this change.